Workbooks/ADXCluster/AtResourceCache/ResourceCacheDetails.workbook

```diff
@@ -565,6 +567,130 @@
               "comparison": "isNotEqualTo"
             },
             "name": "Cache Details per Query Look Back Period",
+            "styleSettings": {
+              "margin": "-50px 0px 0px 0px"
+            }
+                      },
+          {
+            "type": 1,
+            "content": {
+              "json": "<span style=\"font-family: Open Sans; font-weight: 620; font-size: 14px;font-style: bold;margin:-10px -304px 0px 0px;position: relative;top:-5px\">&nbsp;&nbsp;User and query count information per selected query lookback period\r\n\r\n"
+            },
+            "conditionalVisibility": {
+              "parameterName": "DatabaseName",
+              "comparison": "isNotEqualTo"
+            },
+            "name": "Cache Header - Copy"
+          },
+          {
+            "type": 3,
+            "content": {
+              "version": "KqlItem/1.0",
+              "query": "let defaultRetention = 365d * 10;\r\nADXTableUsageStatistics \r\n    | where DatabaseName == '{DatabaseName}' \r\n    | where TableName == '{TableName}' \r\n    | extend NumberOfDaysBefore = datetime_diff('day', StartedOn, MinCreatedOn)\r\n    | where NumberOfDaysBefore == '{LookbackPeriod}'\r\n    | summarize Count=count() by User, ApplicationName\r\n    | top 10 by Count\r\n",
```

Comment:
I think we need to filter on timeRange as well. In other queries I added the timeRange from the parameter(It is only inserted today so not in production yet) WDYT?

Reply:
@svilenko - yes.. It should be filtered the parameter
@radennis - for the alignment, you can use the "Text" column rendered property
![image](https://user-images.githubusercontent.com/45633598/114039347-55113800-988b-11eb-8e32-ccdcd42cca78.png)

Reply:
and please change "LookbackPeriod" to "lookback period"
![image](https://user-images.githubusercontent.com/45633598/114044508-f69a8880-988f-11eb-9544-4c91352716d8.png)

Reply:
Why do we need to filter on timeRange if it's already filtered by the default retention?

Reply:
Maybe I am not looking correctly but I do not see filter by default retention in your query. Regardless, this query is running as a result of choosing a row in the table above. the table above is generated with the timeRange filter. If you do not filter this query as well we might get conflicting results which will be confusing for users.

Reply:
Also, we want all queries to be filtered somehow on timerange so the query would not be heavy